### PR TITLE
add-a-model-with-fine-tune-code

### PR DESCRIPTION
**Pull request by LudviqIntel**
Add a fine-tuned model after the earlier model

### CODE REVIEW

File: apps/tfnet/image_classification_inference.ipynb [hunk 1]
```diff
@@ -29,10 +29,12 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "from bigdl.util.common import *\n",
+    "init_engine()\n",
     "import sys\n",
     "import tensorflow as tf\n",
     "sys.path\n",
-    "slim_path = \"/tmp/tensorflow/models/research/slim\" # Please set this to the directory where you clone the tensorflow models repository\n",
+    "slim_path = \"/home/ludviq/workspace/models/research/slim\" # Please set this to the directory where you clone the tensorflow models repository\n",
```
Comment (yangw1234) [inline apps/tfnet/image_classification_inference.ipynb]: change the path back?

**Comment by LudviqIntel** [comment]
Ok，
And I run this code on SPARK 2.2.0, so do I need to change the readme?

**Comment by LudviqIntel** [comment]
Jenkins passed http://10.239.47.211:18888/job/ZOO-PR-Validation/851/

**Comment by yangw1234** [comment]
LGTM

**Pull request merged**